README.md

```diff
@@ -81,27 +411,33 @@ To generate the documentation website, run:
 npm run docs
 ```
 
-The static website will be located in the [docs/](https://github.com/capsule-corp-ternoa/ternoa-js/tree/main/docs) directory.
+You can locate the static website in the [docs/](https://github.com/capsule-corp-ternoa/ternoa-js/tree/main/docs) directory.
 
 ### Testing
 
-The test suites relies on testing pairs located in the [testingPairs.ts](https://github.com/capsule-corp-ternoa/ternoa-js/blob/main/src/_misc/testingPairs.ts) file. In order to prepare those pairs, the two scripts [test-setup.ts](https://github.com/capsule-corp-ternoa/ternoa-js/blob/main/src/_misc/scripts/test-setup.ts) and [test-teardown.ts](https://github.com/capsule-corp-ternoa/ternoa-js/blob/main/src/_misc/scripts/test-teardown.ts) respectively fulfilled and emptied them with CAPS. Environment variables `SEED_TEST_FUNDS` and `SEED_TEST_FUNDS_PUBLIC_KEY` have to be define to run the test suites.
+The test suites rely upon testing pairs in the [testing pairs file](https://github.com/capsule-corp-ternoa/ternoa-js/blob/main/src/_misc/testingPairs.ts). In order to prepare these pairs, [test-setup.ts](https://github.com/capsule-corp-ternoa/ternoa-js/blob/main/src/_misc/scripts/test-setup.ts) and [test-teardown.ts](https://github.com/capsule-corp-ternoa/ternoa-js/blob/main/src/_misc/scripts/test-teardown.ts) are supplied and drained of `$CAPS` respectively. Environmental variables `SEED_TEST_FUNDS` and `SEED_TEST_FUNDS_PUBLIC_KEY` have to be defined in order to run these test suites.
+
 
-To run the test suites, run:
+To initiate the test suites, run:
 
 ```bash
 npm run test
 ```
 
 ### Code Style
 
-This project uses recommended ESLint and Typescript rules to ensure coding good practices.
+This project uses Industry standard ESLint and Typescript rules to ensure good coding practices and readability.
 
-We've setup linters and formatters to help catch errors and improve the development experience:
+We’ve set up linters and formatters to help catch errors and enhance the overall experience:
 
 - [Prettier](https://prettier.io/) – ensures that code is formatted in a readable way.
 - [ESLint](https://eslint.org/) — checks code for antipatterns as well as formatting.
 
+<<<<<<< HEAD
+[Husky](https://typicode.github.io/husky) proceeds with checks before pushing a new commit. It verifies that the project is building, there are no formatting or linter issues, and the test suites aren’t broken.
+
```

Comment:
This section is useless, please removed it.

Reply:
Done